export/light.py

```diff
@@ -51,7 +51,7 @@ def convert_luxcore_settings(exporter, obj, obj_key, depsgraph, luxcore_scene, t
     # Common light settings shared by all light types
     # Note: these variables are also passed to the area light export function
     gain, importance, lightgroup_id = _convert_common_props(exporter, scene, light)
-    definitions["gain"] = gain
+    definitions["gain"] = [x * pow(2, light.luxcore.exposure) for x in gain]
```

Comment:
This line is duplicated a few times, I think it would be good if you could pull it out into a function like
```python
def apply_exposure(gain, exposure):
    return [x * pow(2, exposure) for x in gain]
```

Reply:
Thanks for the feedback. It's corrected now.